packages/zosfiles/__tests__/__integration__/cli/hRecall/ds/__snapshots__/cli.files.hRecall.ds.integration.test.ts.snap

```diff
@@ -30,8 +30,8 @@ exports[`Recall Data Set should display the help 1`] = `
 
    --wait  | -w (boolean)
 
-      If true then the function waits for completion of the request. If false the
-      request is queued.
+      If true then the function waits for completion of the request. If false
+      (default) the request is queued.
 
```

Comment:
Thanks for documenting what the default behavior is.
Wondering if you had a chance to look at the `defaultValue` in the `ICommandOptionDefinition`.

No change requested since we also have done this both ways in the past :yum:

Reply:
I just had a look at `defaultValue` now! Didn't know this was available, but thanks for letting me know about it!

Reply:
I can add this in to the `mds.definition.ts`, as well as the respective `ds.definition.ts` files for `migrate` and `recall`, only a minor change but its good for consistency, what do you think?

Reply:
It's really a nice to have, and I don't have a strong opinion of one over the other :yum:
It's still very clear what the default behavior is based on the description of the option.
I think ideally we want all options to be defined with their `defaultValue` if applicable.

Reply:
I agree, I will make this change now and push it 😄